bindata/assets/config/defaultconfig.yaml

```diff
@@ -29,7 +29,7 @@ extendedArguments:
   cluster-signing-duration:
   - "720h"
   secure-port:
-  - "10257"
+  - "10358"
```

Comment:
since prometheus is not running in bootstrap phase it probably doesn't make sense to run the proxy there. One thing that is a bit off is that the kube-controller-manager still has the port 10257 in a bootstrap phase and after that proxy overtakes it

Reply:
yeah, 10257 is the default port, can there be more consequences with this change?


```
--secure-port int     Default: 10257
```

Reply:
by the way, where do you set the `--secure-port` option in kcm to listen on port 10358?

Reply:
oh, it is generated when it renders based on the manifest?